### DISCUSSION TITLE
how to use recordException function?

### DISCUSSION THREAD

**Discussion by lowkeyboard** [invertase/react-native-firebase#5350] [Q&A] [answered]
`try {
    methodThatThrows();
} catch (Exception e) {
    FirebaseCrashlytics.getInstance().recordException(e);
    // ...handle the exception.
}`

the code from google docs shows how to record a non-fatal crash report and i want to achieve the same but i couldn't find it in the react-native-firebase reference api.

**Comment by mikehardy** [comment, marked as answer]
Hi there!
https://rnfirebase.io/reference/crashlytics#recordError

**Reply by mikehardy** [reply]
You also might like https://invertase.io/blog/react-native-firebase-crashlytics-configuration
